sdk/resourcemanager/Azure.ResourceManager/src/Resources/Custom/ResourceGroupResource.cs

```diff
@@ -104,5 +104,25 @@ Page<GenericResource> NextPageFunc(string nextLink, int? pageSizeHint)
             }
             return PageableHelpers.CreateEnumerable(FirstPageFunc, NextPageFunc);
         }
+
+        /// RequestPath: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/resources
+        /// ContextualPath: /subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}
+        /// OperationId: Resources_ListByResourceGroup
+        /// <summary> Get all the resources for a resource group. </summary>
+        /// <returns> An async collection of <see cref="GenericResource" /> that may take multiple service requests to iterate over. </returns>
+        public virtual AsyncPageable<GenericResource> GetGenericResourcesAsync()
```

Comment:
These overloads don't follow the pattern [here](https://github.com/Azure/autorest.csharp/issues/1630#issuecomment-1018265506).